src/STRApplication/STRCIController.fs

```diff
@@ -15,7 +15,14 @@ open STRService.Models
 
 open FSharp.Data
 
-type TemplateController () =        
+open STRClient
+
+type STRCIController () =        
+
+    member this.Client =
```

Comment:
If you must keep this as `this.Client` instead of `this.Client()`, this is a reason to use `STRCIController()` instead of `STRCIController`